packages/components/src/RadioGroup/RadioGroup.mdx

```diff
@@ -33,6 +34,30 @@ import { RadioGroup } from "@jobber/components/RadioGroup";
   }}
 </Playground>
 
-## Props
+## RadioGroup Props
 
 <Props of={RadioGroup} />
+
+## RadioOption Props
+
+<Props of={RadioOption} />
+
+## Disabled Options
+
+`RadioOptions` can be disabled by passing in the disabled prop. Disabling an
+option will prevent a user from being able to select the option.
+
```

Comment:
I wonder if it is worth mentioning that this can be set dynamically, not just when the form loads....

Reply:
It would be quite unusual for a prop to not allow this. I'm not sure it's necessary…?

Reply:
I will make the playground a little better for this.